api/src/pdc/providers/superstruct/shared/index.ts

```
@@ -65,3 +66,11 @@ export const Siren = pattern(string(), /^[0-9]{9}$/);
 export const Siret = pattern(string(), /^[0-9]{14}$/);
 export const TerritoryCode = union([Country, Department, Insee, Siren]);
 export const TerritoryType = enums(["com", "epci", "aom", "dep", "reg", "country"]);
+export const Boolean = coerce(boolean(), string(), (v) => {
```

Biome (1.9.4)

[error] Do not shadow the global "Boolean" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)